Model/Sales/TrackTraceHolder.php

```diff
@@ -36,6 +36,8 @@ class TrackTraceHolder
      */
     const MYPARCEL_TRACK_TITLE  = 'MyParcel';
     const MYPARCEL_CARRIER_CODE = 'myparcelnl';
+    const ORDER_NUMBER          = '[ORDER_NR]';
```

Comment:
```suggestion
    const ORDER_NUMBER_PLACEHOLDER          = '%order_nr%';
```

Het is gebruikenlijker om {order_nr} of %order_nr% te gebruiken voor een placeholder

Reply:
Done